docs/articles/auth-install.md

```diff
@@ -27,22 +27,18 @@ Let's assume that we need to setup email & password authentication based on Nebu
    // ...
     
    NbAuthModule.forRoot({
-         providers: {
-           email: {
-             service: NbEmailPassAuthProvider,
-             config: {
-              ...
-             },
-           },
-         },
-         forms: {},
-       }), 
+     providers: [
+       NbEmailPassAuthProvider.register('email', { ... }),
```

Comment:
Do you really think we have to duplication **email** in the name of the provider and in the first param of the *register* function? Maybe we can remove this duplication somehow?